docs/supportedsites.md

```diff
@@ -115,6 +116,7 @@
  - **CNN**
  - **CNNArticle**
  - **CNNBlogs**
+ - **cntv**: 央视网
```

Comment:
oops...because i use Makefile in youtube-dl, so it auto added to the file.
I will clean it.